server/src/main/java/org/elasticsearch/search/retriever/RetrieverBuilder.java

```diff
@@ -93,7 +93,7 @@ public <T> T namedObject(Class<T> categoryClass, String name, Object context) th
         return parseInnerRetrieverBuilder(parser, context);
     }
 
-    protected static RetrieverBuilder parseInnerRetrieverBuilder(XContentParser parser, RetrieverParserContext context) throws IOException {
+    public static RetrieverBuilder parseInnerRetrieverBuilder(XContentParser parser, RetrieverParserContext context) throws IOException {
```

Comment:
Why the change in visibility? 🤔 I can't see it referenced outside this class

Reply:
No reason 😅  Probably leftover of some initial refactoring/testing